### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nathandroberts**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    oTurn = false;
+    xTurn = true;
+    whatPlayer();
+    winnerCheck();
+  }
+});
+function winnerCheck() {
+  
+  if (winConditons.indexOf(xMoves) > -1){
+    alert('winner x')
```
Comment (JSChilds) [inline script.js]: Is there a better way you could get this information to the player than an alert?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  xTurn = true;
+  oTurn = false;
+  whatPlayer();
+
+})
+//x o grid inputs
+$('.box').click(function(){
+  if (xTurn === true && ($(this).attr('class') === 'box'|| $(this).attr('class') ==='box clear' )) {
+  //add X class and x to box
+  $(this).addClass('X').html('X')
```
Comment (JSChilds) [inline script.js]: This should be indented.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Very close and your code is clear and readable. Consider using what you learned in the homework review to complete this.

**Comment by nathandroberts** [comment]
Winner now displayed in text rather than as an alert

**Review by JSChilds — APPROVED** [review]

**Pull request merged**